static/src/javascripts/bootstraps/enhanced/notifications.js

```diff
@@ -107,9 +107,10 @@ define([
         },
 
         subscribeHandler: function () {
+            var alreadyGranted = Notification.permission === 'granted';
             modules.subscribe().then(modules.follow)
                 .then(function() {
-                    if (Notification.permission === 'granted') {
+                    if (!alreadyGranted && Notification.permission === 'granted') {
```

Comment:
it does make this code a bit of a weird read,
I had a quick look if there's an event that's fired when `Notification.permission` changes, but there doesn't seem to be one. 
I'd probably just add a comment around here that explains it gets changed at some point during `modules.subscribe` or `modules.follow`